tests/topotests/ospf6_topo2/test_ospf6_topo2.py

```diff
@@ -334,7 +334,7 @@ def test_nssa_lsa_type7():
     route = {
         "2001:db8:100::/64": {
             "pathType": "E2",
-            "nextHops": [{"nextHop": "::", "interfaceName": "r4-eth0"}],
+            "nextHops": [{"interfaceName": "r4-eth0"}],
```

Comment:
The output of the route display changed: it now points to a link-local address instead of nothing (see PR initial post). I think this should be in a separated commit with an appropriated description.